report.go

```diff
@@ -22,5 +27,15 @@ func MakeReport() (Report, error) {
 		return Report{}, err
 	}
 	report := Report{ram, swap, os, cpu, interfaces}
+
 	return report, err
 }
+
+func LogWriter() {
```

Comment:
make this multi purpose.  So, sorta a big change in direction...

Make this a method call for each type that we have create so far.  That way, each type can log to syslog how it needs to.